packages/cli/src/commands/lockedgold/lock.ts

```diff
@@ -28,14 +28,18 @@ export default class Lock extends BaseCommand {
 
     this.kit.defaultAccount = address
     const value = new BigNumber(res.flags.value)
+
+    await newCheckBuilder(this)
+      .addCheck(`Value [${value.toFixed()}] is >= 0`, () => value.gt(0))
```

Comment:
Nitpicky, but text says >= 0 while code is gt, not gte (assuming gte is an available function).
Unfamiliar with the greater context here so I'm not sure if this is at all significant. I'm assuming not

Reply:
Whoops, nice catch, fixed.